math/antic/Portfile

```diff
@@ -1,34 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
+PortGroup           obsolete 1.0
 
-github.setup        flintlib antic ba3b457281f492c7f9680247fb2c98208ecfa69c
+name                antic
 version             2022.11.30
-revision            0
+revision            1
 categories          math
 license             LGPL-2.1
-maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
-description         Algebraic Number Theory In C
-long_description    {*}${description}
-checksums           rmd160  2c2b8923b9a9826ba4864f09837a3696cf1cdcdc \
-                    sha256  e1b91e97823cca4fdb1a8932614a2a1f60780a562370d1c06710666e56de0aae \
-                    size    80398
-
-depends_lib-append  port:flint \
-                    port:gmp \
-                    port:mpfr
-
-post-patch {
-    reinplace "s|/usr/local|${prefix}|g" ${worksrcpath}/configure
-}
-
-compiler.thread_local_storage yes
-
-platform darwin 10 powerpc {
-    configure.args-append \
-                    --build=ppc-Darwin
-}
-
-test.run            yes
-test.target         check
+# https://github.com/flintlib/antic/issues/83
+replaced_by         flint
```

Comment:
please add a comment as to when the port can be deleted (i.e., one year from now).

Reply:
Done